xbmc/commons/ilog.h

```diff
@@ -45,3 +45,4 @@ constexpr int LOGEPG = (1 << (LOGMASKBIT + 16));
 constexpr int LOGANNOUNCE = (1 << (LOGMASKBIT + 17));
 constexpr int LOGWSDISCOVERY = (1 << (LOGMASKBIT + 18));
 constexpr int LOGADDONS = (1 << (LOGMASKBIT + 19));
+constexpr int LOGGRAPHICS = (1 << (LOGMASKBIT + 20));
```

Comment:
I not think new category is necessary. `LOGVIDEO` already exist for these things and is almost not used in Windows.

Reply:
fwik LOGVIDEO is used by VP, for me it is no problem to reuse the existing one,
maybe a third evaluation by someone else would be nice

Reply:
It's used every frame by vp but for addons only it seems. For internal it barely logs anything when playing (I tried). Reusing will work with things as they are now.